site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1673,6 +1673,47 @@ public function getAverageForGradeable($g_id, $section_key, $is_team) {
         }
         return new SimpleStat($this->core, $this->course_db->rows()[0]);
     }
+    
```

Comment:
Can you add PHP type hints to all of these functions, it will make static analysis perform better wherever they are used.

Reply:
Added the PHP type hints